www/components/ModuleListSidebar.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { Box } from "@welcome-ui/box";
+import { Text } from "@welcome-ui/text";
+import { Link } from "@welcome-ui/link";
+import styled from "styled-components";
+
+import { Pkg } from "../lib/docs";
+
+const ModuleLink = styled(Link)<{ active: boolean }>`
+  padding-left: ${(props) => props.theme.space.xs};
```

Comment:
Ideally *all* spacing in CSS would be using values from the theme, for visual consistency.